### PR TITLE
12925 fss UI simplified search screen reader

### PR DESCRIPTION
**Pull request by gaurav15233**
This PR contains FSS UI simplified search screen reader implementation changes and this PR has dependency of PR #118

### CODE REVIEW

File: src/app/features/fss-search/fss-search-results/fss-search-results.component.ts [hunk 1]
```diff
@@ -21,15 +22,17 @@ export class FssSearchResultsComponent implements OnChanges {
   ngOnChanges(): void {
     this.searchResultVM = [];
     if (this.searchResult.length > 0) {
+      let currentPage = this.currentPage; 
       var batches = this.searchResult[0];
-      for (var i = 0; i < batches.length; i++) {
+      for (var i = 0, SrNo = 1; i < batches.length; i++, SrNo++) {
```
Comment (crdandoukho) [inline src/app/features/fss-search/fss-search-results/fss-search-results.component.ts:27]: Please change the capitalisation of SrNo to camel-case to match the convention used for other variables

File: src/app/features/fss-search/fss-search.component.html [hunk 2]
```diff
@@ -33,13 +33,13 @@ <h3>{{messageTitle}}</h3>
...
-      <div >
-        <h2>
+      <div>
+        <h2 id="skipResultsLink" tabindex="-1">
           <span class="totalResultLabel" *ngIf="searchResultTotal == 1" role="alert">{{this.searchResultTotal}} result found</span>
           <span class="totalResultLabel" *ngIf="searchResultTotal > 1" role="alert">{{this.searchResultTotal}} results found</span>
           <span #showSearchResult *ngIf="searchResultTotal > pageRecordCount" class="paginatorLabel" role="alert"> {{this.paginatorLabel}}</span>
         </h2>
-        <app-fss-search-results class="SimplifiedSearchResult" id="skipResultsLink" tabindex="-1" [searchResult]="searchResult"
+        <app-fss-search-results class="SimplifiedSearchResult" [searchResult]="searchResult" [currentPage] = "currentPage"
```
Comment (crdandoukho) [inline src/app/features/fss-search/fss-search.component.html:42]: Please remove the spaces around the equals
```suggestion
        <app-fss-search-results class="SimplifiedSearchResult" [searchResult]="searchResult" [currentPage]="currentPage"
```

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Looks good, just a few minor changes needed please

File: src/app/features/fss-search/fss-search-results/fss-search-results.component.ts (hunk 1 above)
Comment (gaurav15233) [reply src/app/features/fss-search/fss-search-results/fss-search-results.component.ts]: updated as suggested. Thanks

File: src/app/features/fss-search/fss-search.component.html (hunk 2 above)
Comment (gaurav15233) [reply src/app/features/fss-search/fss-search.component.html:42]: updated as suggested. Thanks

**Pull request merged**